tests/unit-tests/test-class-sensei-data-cleaner.php

```diff
@@ -432,4 +494,36 @@ public function testSenseiRolesAndCapsDeleted() {
 		$role = get_role( 'teacher' );
 		$this->assertNull( $role, 'Teacher role should be removed overall' );
 	}
+
+	/**
+	 * Ensure comments are trashed.
+	 *
+	 * @covers Sensei_Data_Cleaner::cleanup_all
+	 * @covers Sensei_Data_Cleaner::cleanup_comments
+	 */
+	public function testCleanupComments() {
+		$this->setupComments();
+
+		Sensei_Data_Cleaner::cleanup_all();
+
+		foreach ( $this->comment_ids as $comment_id ) {
```

Comment:
You can also test comment meta if you want. You can see how I did it for post meta in [this PR](https://github.com/Automattic/sensei/pull/2105). But since removing the meta is built-in WP functionality, probably it's not necessary to test. I'm just paranoid 😄